frontend/tickets-online/src/hall/components/PlaceInfo.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import '../../bootstrap.css';
+import '../../index.css';
+
+class PlaceInfo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick(e) {
+    e.preventDefault();
+    this.props.onBookBtnClick();
+  }
+
+  render() {
+    return (
+      <div className="row justify-content-md-center place-info">
+        <div className="alert alert-light text-center">
+          <h4>Place Information</h4>
+          Row: {this.props.rowNumber}
+          <br />
```

Comment:
It is a bad approach to use <br /> tag to add margins between items. Html has to be semantically correct. If you use <br/> it means that between items always should be line. I am not sure that it is this case. Only CSS has to be used for styling. Let's discuss if it is not clear